src/autoscaler.ts

```diff
@@ -0,0 +1,54 @@
+import * as dotenv from 'dotenv';
+
+import { JibriTracker, JibriStatusState } from './jibri_tracker';
+
+import logger from './logger';
+
+dotenv.config();
```

Comment:
I think it makes more sense to have the env config in the main app.ts and then either put all the config in a module or there. I split it in the queue app because I am trying to keep separated code I think is re-usable by the community and things that are specific to our install. In this case I think all config belongs together for simplicity of finding it later.